### ISSUE TITLE
Not changing anything in Chrome://Discards

### ISSUE THREAD

**Issue by syedshariq121** [brian-girko/always-active#9]
Does this change status in Chrome://Discards? as for me it is still showing hidden in status tab.

**Comment by brian-girko** [comment]
No, this internal page displays real properties and this extension cannot alter them. The extension only tries to spoof web pages to hide the visibility properties and events from the page-level scripts.
